stable/auth/v1/auth.proto

```diff
@@ -7,7 +7,12 @@ package protocol.auth.v1;
 option go_package = "github.com/harmony-development/legato/gen/auth/v1";
 
 // Used in `BeginAuth` endpoint.
-message BeginAuthRequest {}
+message BeginAuthRequest {
+  // The guest token for an account that will be upgraded
+  // from guest to full status by successful completion
+  // of the auth process.
+  optional string for_guest_token = 1;
```

Comment:
it isn't clear that this is the guest token that has already been generated by the server in a previous auth session